moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -418,22 +425,22 @@ bool IKFastKinematicsPlugin::initialize(const rclcpp::Node::SharedPtr& node,
 
   storeValues(robot_model, group_name, base_frame, tip_frames, search_discretization);
 
-  RCLCPP_INFO_STREAM(LOGGER, "Using link_prefix: '" << params_.link_prefix << '\'');
+  RCLCPP_INFO_STREAM(getLogger(), "Using link_prefix: '" << params_.link_prefix << '\'');
 
   // verbose error output. subsequent checks in computeRelativeTransform return false then
   if (!robot_model.hasLinkModel(tip_frames_[0]))
-    RCLCPP_ERROR_STREAM(LOGGER, "tip frame '" << tip_frames_[0] << "' does not exist.");
+    RCLCPP_ERROR_STREAM(getLogger(), "tip frame '" << tip_frames_[0] << "' does not exist.");
   if (!robot_model.hasLinkModel(base_frame_))
-    RCLCPP_ERROR_STREAM(LOGGER, "base_frame '" << base_frame_ << "' does not exist.");
+    RCLCPP_ERROR_STREAM(getLogger(), "base_frame '" << base_frame_ << "' does not exist.");
 
   if (!robot_model.hasLinkModel(params_.link_prefix + IKFAST_TIP_FRAME_))
-    RCLCPP_ERROR_STREAM(LOGGER, "prefixed tip frame '" << params_.link_prefix + IKFAST_TIP_FRAME_
-                                                       << "' does not exist. "
-                                                          "Please check your link_prefix parameter.");
+    RCLCPP_ERROR_STREAM(getLogger(), "prefixed tip frame '" << params_.link_prefix + IKFAST_TIP_FRAME_
+                                                            << "' does not exist. "
```

Comment:
Was this added by the auto-format?

Reply:
yes, `getLogger()` is more characters than `LOGGER` so the line is reformated.